docs: Document Actor reviews (#2580)
apify/apify-docs: sources/platform/actors/publishing/actor_rating.mdx

---
title: Monitor your Actor's rating
description: Learn how to monitor your Actor's rating in Apify Store to ensure it meets the quality standards and is performing well.
slug: /actors/publishing/actor-rating
sidebar_position: 6
---

Users can review your Actor in Apify Store. Their ratings impact your Actor's [quality score](/platform/actors/publishing/quality-score) and help it rank higher in search results. User reviews are also a great source of feedback that you can use to improve your Actor.

## About the Actor rating

> Your Actor's rating isn't a pure average of all reviews. Instead, it's a weighted average that includes the following factors:

Reviewer: Doesn't it make more sense to lead in with what it is? Or is the weighted average an actual assumption that community has and we want to dispel that explicitly?

```suggestion
Your Actor's rating is a weighted average that includes the following factors:
```

- More recent reviews have a higher weight than the older ones.
- Reviews from trusted users have a higher weight.

> - Reviews from users who often run your Actor have a higher weight.

Reviewer 2: I would maybe skip that line since it can lead to better review fraud 😅
